components/blog/index.tsx

```diff
@@ -7,6 +7,7 @@ import { format } from "date-fns";
 import { InlineEditorContext } from "context/InlineEditorContext";
 import InlineEditor from "components/InlineEditor";
 import { MyPortableTextComponents, BlogsData } from "types";
+import { defaultBlockStyle } from "helper";
```

Comment:
Hello! Do we intend to use this import for the block style of the blog component? 
If not being used, I think it would be best to remove it to avoid confusion with the block style template defined in line 15. Thanks!